portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,35 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    private ArrayList<String> comments;
```

Comment:
You can define this list inside doGet to make it clear what this list is for. Since you recreate the list on each request, there isn't any need to define this as a class variable.

(an alternative and better option would be to create the list as a static final class variable so that its only ever created once!)